### PR TITLE
Added signal for Tween emitted at full completion

### LINKED ISSUE #27220: Tween please add a Signal named tween_all_completed
**Godot version:**
3.1 stable

**OS/device including version:**
ALL

**Issue description:**

when i use tween, i have 4 interpolate_property

```
	twPath.remove_all()

	twPath.interpolate_property(m1,"rect_scale", m1.rect_scale, Vector(1,1), time2, Tween.TRANS_SINE , Tween.EASE_IN)
	twPath.interpolate_property(m2,"rect_scale", m2.rect_scale,  Vector(1,1), time2, Tween.TRANS_SINE , Tween.EASE_IN)
	twPath.interpolate_property(m1,"rect_position", m1.position, Vector2(0,0), time2, Tween.TRANS_SINE , Tween.EASE_IN)
	twPath.interpolate_property(m2,"rect_position", m2.position, Vector2(60,0), time2, Tween.TRANS_SINE , Tween.EASE_IN)

	twPath.start()

	yield(twPath,"tween_completed")
	yield(twPath,"tween_completed")
	yield(twPath,"tween_completed")
	yield(twPath,"tween_completed")

	# do something clean work
```

i must add yield  4 times when i want to judge the all 4 interpolate_property complete

please  add a Signal named "tween_all_completed", so I only add
1 yield  instead at the end.

`yield(twPath,"tween_all_completed")`

### PR DESCRIPTION
**Pull request by Chaosus**
Fix #27220

### CODE REVIEW

**Comment by girng** [comment]
freaking awesome :D

File: doc/classes/Tween.xml [hunk 1]
```diff
@@ -350,6 +350,11 @@
 		</member>
 	</members>
 	<signals>
+		<signal name="tween_all_completed">
+			<description>
+				Emitted when all processes in a tween ends.
```
Comment (akien-mga) [inline doc/classes/Tween.xml]: Should be "end" here for plural, not "ends".

Comment (Chaosus) [reply doc/classes/Tween.xml]: Fixed

**Comment by reduz** [comment]
seems ok

**Comment by akien-mga** [comment]
Thanks!

**Comment by hpvb** [comment]
Cherry-picked for 3.1.1.

**Pull request merged**